special-pages/shared/components/Switch/Switch.js

```diff
@@ -1,22 +1,20 @@
 import { h } from 'preact';
 import styles from './Switch.module.css';
-import { useEnv } from '../../../../shared/components/EnvironmentProvider';
 
 /**
  * Switch component used to toggle between two states.
  *
  * @param {Object} props - The props object.
  * @param {string} props.ariaLabel - Label applied to the <input />
- * @param {ImportMeta['injectName']} [props.variant]
+ * @param {"macos" | "windows"} props.platformName - only supported platforms
+ * @param {"light" | "dark"} props.theme
```

Comment:
now `platformName` + `theme` are required. This allows a single page to render the switch in either or both.